scripts/travis/build_images.sh

```diff
@@ -14,7 +14,7 @@
 
 # Pull base image quietly.  We might want to use
 # tensorflow/tensorflow:2.1.0-gpu-py3 for GPU-accelerated deep learning.
-BASE_IMAGE=tensorflow/tensorflow:2.1.0-py3
+BASE_IMAGE=tensorflow/tensorflow:2.1.2
```

Comment:
What is the difference between 2.1.2 and 2.1.0-py3?

Reply:
The pre-installed tensorflow version are different.
CI image building will be much faster if we use 2.1.2 because elasticdl pkg requires 2.1.2